tasks_downstream.py

```diff
@@ -46,9 +46,14 @@
         "build"
     ]["args"]["ODOO_VERSION"]
 )
-DOCKER_COMPOSE_CMD = f"{shutil.which('docker')} compose" or shutil.which(
-    "docker-compose"
+
+docker_compose_result = subprocess.run(
+    [which("docker"), "compose"], capture_output=True, text=True
 )
+if docker_compose_result.returncode == 0:
+    DOCKER_COMPOSE_CMD = "docker compose"
+else:
+    DOCKER_COMPOSE_CMD = "docker-compose"
```

Comment:
You need the whole path to run the commands
```suggestion
DOCKER_COMPOSE_CMD = f"{shutil.which('docker')} compose" if docker_compose_v2 0 else  shutil.which(
    "docker-compose"
```